src/bootstrap/native.rs

```diff
@@ -175,6 +175,14 @@ impl Step for Llvm {
             cfg.define("LLVM_ENABLE_ZLIB", "OFF");
         }
 
+        // Are we compiling for iOS/tvOS?
+        if target.contains("apple") && !target.contains("darwin") {
+            cfg.define("CMAKE_OSX_SYSROOT", "/");
+            cfg.define("CMAKE_OSX_DEPLOYMENT_TARGET", "");
+            cfg.define("LLVM_ENABLE_PLUGINS", "OFF"); // Prevent cmake from adding -bundle to CFLAGS automatically.
```

Comment:
This also seems pretty out there -- `-bundle` being added to CFLAGS and LLVM_ENABLE_PLUGINS feels like they're pretty orthogonal? Do we have a link to some documentation about what this option does?

Reply:
Ha, there WAS no documentation. I was told to try this by @eddyb, and it worked.

Reply:
As above, just documenting that these are "magic values" we don't fully understand seems fine for a Tier 3 platform. Maybe at some point we'll get a better understanding of what's going on.

Reply:
But `aarch64-apple-ios` is Tier 2?

Reply:
Resolved by https://github.com/rust-lang/rust/pull/73903/commits/4fd1c77a24288383600b02a7e3f11bf5eb718a32